congress/metadata/B001299.yaml

```diff
@@ -0,0 +1,2 @@
+name: Gage Skidmore
+link: https://commons.wikimedia.org/wiki/File:Jim_Banks_by_Gage_Skidmore.jpg
```

Comment:
This has a CC-BY-SA 3.0 license attached, and needs to be replaced with a public domain version.